terraform/modules/rds/database.tf

```diff
@@ -8,21 +7,27 @@ resource "aws_db_instance" "this" {
   #checkov:skip=CKV_AWS_353:performance insights TODO CLDC-2660 if necessary
   #checkov:skip=CKV_AWS_354:performance insights TODO CLDC-2660 if insights are necessary
   #checkov:skip=CKV2_AWS_30:query logging TODO CLDC-2660
-  #checkov:skip=CKV2_AWS_60:copy tags to snapshots TODO CLDC-2661
   identifier                 = var.prefix
   apply_immediately          = true
   auto_minor_version_upgrade = true
   allocated_storage          = var.allocated_storage #units are GiB
+  backup_retention_period    = var.backup_retention_period
+  backup_window              = "23:09-23:39"
+  copy_tags_to_snapshot      = true
   db_name                    = "data_collector"
   db_subnet_group_name       = var.db_subnet_group_name
+  delete_automated_backups   = false
   deletion_protection        = true # needs to be set to false and applied if you need to delete the DB
   engine                     = "postgres"
   engine_version             = "13.11"
+  final_snapshot_identifier  = var.prefix
   instance_class             = var.instance_class
+  maintenance_window         = "Mon:02:33-Mon:03:03"
```

Comment:
I didn't think we'd want different backup and maintenance windows for the different environments, so didn't make this into a variable.